### DISCUSSION TITLE
Changing Input files

### DISCUSSION THREAD

**Discussion by jansteinhauser** [magpiemodel/magpie#364] [Q&A] [answered]
Hello and happy new year!

I used to add/change some input files via a patch folder as described in https://github.com/magpiemodel/tutorials/blob/master/6_AdvancedChangeInputs.md.

But with the latest MAgPIE version this doesn't seem to work anymore. I get the warning
> Settings are unknown in provided cfg (cfg$repositories$./patch_inputdata)! 
and the patch file is simply ignored.

The code in my start script is simply as follows and , of course, the target folder exists:

```
cfg$repositories <- append(list("https://rse.pik-potsdam.de/data/magpie/public"=NULL,
                                "./patch_inputdata"=NULL),
                           getOption("magpie_repos"))
```

Any advice? 
Thanks so much!
Jan

**Comment by tscheypidi** [comment, marked as answer]
Hej there and happy new year to you as well,

the warning message is a false positive which I will look into, but it should not alter your configuration. Might there be another reason why your patch is being ignored?

**Reply by jansteinhauser** [reply]
Indeed there was another - kind of awkward - reason: I didn't tgz the file I wanted to patch but a shortcut to it (ironically, probably because I was a bit lazy and thought it might be quicker to copy in the file explorer than via terminal - definitely didn't save time on that decision) 

So, the lesson today is to check the process from start to finish. Though, to be fair, that warning does send one off in a wrong direction.

Thanks for nudging me to re-check :)

**Reply by tscheypidi** [reply]
I agree that the warning was perfectly suited to get you off track. PR https://github.com/magpiemodel/magpie/pull/365 will make sure that the warning does not pop up anymore in these instances
